setup.py

```diff
@@ -25,10 +25,13 @@
 # override these and pass --static for a static build. See
 # doc/build.txt for more information. If you do not pass --static
 # changing this will have no effect.
-STATIC_INCLUDE_DIRS = []
-STATIC_LIBRARY_DIRS = []
+def static_env_list(name):
+    return list(filter(None, map(lambda x: x.strip(), os.environ.get(name, "").split(","))))
+
+STATIC_INCLUDE_DIRS = static_env_list("STATIC_INCLUDE_DIRS")
+STATIC_LIBRARY_DIRS = static_env_list("STATIC_LIBRARY_DIRS")
 STATIC_CFLAGS = []
-STATIC_BINARIES = []
+STATIC_BINARIES = static_env_list("STATIC_BINARIES")
```

Comment:
Wouldn't it be better to make the env variable names lxml specific, i.e. `LXML_STATIC_BINARIES` etc.?
Otherwise, they could be accidentally picked up from somewhere else in a larger dependency build.